TUM-Dev/NavigaTUM
# Refoldering

## Proposed Changes (include Screenshots if possible)

- This PR refactors every microservice to live under the `server` folder.

## How to test this PR

1. See if this is a good change
2. Testing in production if the simplified deployment system holds its ground.

## How has this been tested?

- internal monolouge

## Checklist:
